cluster/definition.go

```diff
@@ -132,8 +148,8 @@ type Definition struct {
 	// Creator identifies the creator of a cluster definition. They may also be an operator.
 	Creator Creator `json:"creator" ssz:"Composite" config_hash:"9" definition_hash:"9"`
 
-	// ValidatorAddresses define addresses of each validator.
-	ValidatorAddresses []ValidatorAddresses `json:"validators" ssz:"CompositeList[65536]" config_hash:"10" definition_hash:"10"`
+	// Validators define addresses of each validator.
+	Validators []ValidatorAddresses `json:"validators" ssz:"CompositeList[65536]" config_hash:"10" definition_hash:"10"`
```

Comment:
why rename, lets keep it as "addresses" until it contains something else?

Reply:
okay cool I'll revert this one